00-Introducing-HOOMD-blue/03-Initializing-the-System-State.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<p>I might not put any <strong>initial condition</strong> as there can be metastable states.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It's a bit pedantic - but as long as the system is ergodic, any initial condition will eventually reach the free energy minimum state. In this case, I'll just remove the "any initial condition" as it is not necessary to convey the intended description of self-assembly.</p>

Reply:
That is true given ergodicity, any initial condition will reach equilibrium.